reana_commons/config.py

```diff
@@ -43,10 +43,18 @@
 """RabbitMQ connection string."""
 
 STATUS_QUEUE = 'jobs-status'
-"""Name of the queue where to publish/consume from."""
 
 EXCHANGE = ''
-"""RabbitMQ exchange."""
 
 ROUTING_KEY = 'jobs-status'
+
+MQ_DEFAULT_SERIALIZER = 'json'
+"""Default serializing format (to consume/produce)."""
+MQ_DEFAULT_EXCHANGE = ''
+"""RabbitMQ exchange."""
+MQ_DEFAULT_QUEUE = 'jobs-status'
+"""Name of the queue where to publish/consume from."""
+MQ_DEFAULT_ROUTING_KEY = 'jobs-status'
 """RabbitMQ routing key."""
+MQ_PRODUCER_MAX_RETRIES = 3
+"""Max retries to send a message."""
```

Comment:
We refer to RabbitMQ in this file both as `BROKER_...` and `MQ_...`, maybe we could use one term. There are the configs for the message broker service, and the configs for the messages and message queues tho that we could distinguish between.